src/views/Home.vue

```diff
@@ -7,7 +7,7 @@
             <div class="card-header">
               <span>JWT Token</span>
               <span>
-                <el-button size="small" @click="fetchJWT()"> Fetch </el-button>
+                <el-button size="small" @click="getAttestation()"> Fetch </el-button>
```

Comment:
@longtomjr should we remove the "validate" button that was previously getting the attestation?